test/addons/openssl-binding/binding.gyp

```diff
@@ -1,11 +1,22 @@
 {
+  'includes': ['../../../config.gypi'],
+  'variables': {
+    'node_target_type%': '',
+  },
   'targets': [
     {
       'target_name': 'binding',
       'conditions': [
          ['node_use_openssl=="true"', {
            'sources': ['binding.cc'],
            'include_dirs': ['../../../deps/openssl/openssl/include'],
+           'conditions': [
+              ['OS=="win" and node_target_type=="static_library"', {
+	        'libraries': [
+                  '../../../../$(Configuration)/lib/<(OPENSSL_PRODUCT).lib'
```

Comment:
You can drop the `.lib` here now, right?

Reply:
Nit: Could you fix the indentation?
and ditto on the next file (test/addons/zlib-binding/binding.gyp)

Reply:
Ah yes, will fix that now and remove those variables. Thanks